docs/metrics/coefficient-of-determination.md

```diff
@@ -0,0 +1,78 @@
+# Coefficient of Determination (R²)
+
+The Coefficient of Determination, commonly denoted as R², is a statistical measure used to assess the goodness of fit
+of a regression model. It represents the proportion of the variance in the model inferences that is
+explainable by the model itself.
```

Comment:
nit: should this be "explained"?

Reply:
I think both work - chose explainable so that it is kind of future-tense-y as well haha.